bakalaurinis.tex

```diff
@@ -0,0 +1,99 @@
+\usepackage{hyperref}  % Nuorodų aktyvavimas
+\usepackage{listings}
+\usepackage{subfig}
+\usepackage{url}
+\usepackage{wrapfig}
+
+
+% Titulinio aprašas
+\university{Vilniaus universitetas}
+\faculty{Matematikos ir informatikos fakultetas}
+% \institute{Informatikos institutas}  % Užkomentavus šią eilutę - institutas neįtraukiamas į titulinį
+\department{Programų sistemų katedra}
+\title{Daugiaplatformių mobiliųjų aplikacijų kūrimo karkasų analizė}
+\titleineng{Analysis of cross-platform mobile app development frameworks}
+\papertype{Bakalaurinio darbo planas}
+\status{4 kurso 4 grupės studentas}
+\author{Edas Lakavičius}
+% \secondauthor{Vardonis Pavardonis}   % Pridėti antrą autorių
+\supervisor{lekt. Jonas Ragaišis}
+\date{Vilnius \\ \the\year}
+
+% Nustatymai
+% \setmainfont{Palemonas}   % Pakeisti teksto šriftą į Palemonas (turi būti įdiegtas sistemoje)
+\bibliography{bibliografija} 
+
+\begin{document}
+\maketitle
+
+\tableofcontents
+
+\section{Tyrimo objektas ir aktualumas}
+Tyrimo objektas - daugiaplatformių mobiliųjų aplikacijų kūrimo karkasai.
+
+Šiais laikais internetas, socialinė medija ir elektroninė prekyba yra neatsiejami nuo kasdienybės dalykai. Interneto naudojimas išaugo nuo 413 klientų milijonų 2000 metais iki 3.4 bilijonų 2016 \cite{owidinternet} ir vis dar auga beprotišku greičiu kiekvieną dieną. Didžioji dalis interneto naudotojų yra per mobilius telefonus prisijungiantys klientai, kurie greit lenks kompiuterius naudojančius klientus, net 4 kartus \cite{internetusage}. Būtent dėl to nenuostabu, kad atsiranda vis daugiau projektų, kurie fokusuojasi į mobiliųjų aplikacijų kūrimą.
+
+Šiuo metu didžiausios aplikacijų parduotuvės "Google Play" ir "Apple App Store" turi daugiau nei 5 milijonų mobiliųjų aplikacijų \cite{appcount} ir įmonei, norinčiai efektyviai dalyvauti šioje rinkoje reiks ne tik daug investuoti, bet ir gebėti greitai kurti ir gerinti savo aplikacijas. Įprastai yra kuriamos atskiros aplikacijos "iOS" ir "Android" platformom, kas dažniausiai reikalauja turėti atskiras komandas ar programuotojus specializuojančius prie vienos iš šių sistemų, tačiau pačios aplikacijos turi būti kuriamos vienodos.
+
+Būtent dėl didelių kaštų, aplikacijų kūrimo greičio ir skirtingų platformų apjungimo išpopuliarėjo hibridinio tipo technologijos - daugiaplatformių mobiliųjų aplikacijų kūrimo karkasai. Per trumpą laiką buvo sukurta gana daug skirtingų technologijų skirtų tam pačiam prieš tai jau minėtam tikslui - apjungti skirtingas platformas. Todėl yra aktualu šiuos karkasus išanalizuoti ir palyginti, kad skirtingi projektai galėtų pasirinkti sau tinkamiausią.
+
+
+\section{Darbo tikslas}
+Darbu siekiama, atliekant skirtingų daugiaplatformių aplikacijų kūrimo karkasų analizę, sukurti rekomendacijų rinkinį, padėsiantį pasirinkti tarp išanalizuotų variantų.
+
+\begin {section}{Keliami uždaviniai ir laukiami rezultatai}
+Keliami uždaviniai:
+\begin{itemize}
+  \item Sukurti bazines aplikacijas, naudojant pasirinktus karkasus.
```

Comment:
Pasirinkti karkasai turėtų atsispindėti įvade (kur dažnai nurodoma problematika) bei galbūt tiksle. Dabar skaitant darbo pirmuosius puslapius vis dar nėra aišku ar tai tas darbas, kurį norėčiau skaityti ar ne. Kai fokusuojamasi, tai yra daugiau konkretumo ir skaitytojas gali atsirinkti kas jam aktualu. Dabar mes dar nieko konkrečiai nepasakom (o tai blogai) ir dėl to tikrai kabinėsis komisija